reframe/core/schedulers/torque.py

```diff
@@ -53,7 +53,9 @@ def poll_jobs(self, jobs):
 
         # Depending on the configuration, completed jobs will remain on the job
         # list for a limited time, or be removed upon completion.
-        # If qstat cannot find the jobid, it returns code 153.
+        # If qstat cannot find any of the jobids, it returns code 153.
+        # Otherwise, it will return with return code 0 and print information
+        # only for the ones it could find.
```

Comment:
```suggestion
        # If qstat cannot find any of the job IDs, it will return 153.
        # Otherwise, it will return 0 and print information
        # only for the jobs it could find.
```